lib/active_record_shards/configuration_parser.rb

```diff
@@ -23,9 +23,14 @@ def explode(conf)
       end
 
       conf.to_a.each do |env_name, env_config|
-        if slave_conf = env_config.delete('slave')
-          expand_child!(env_config, slave_conf)
-          conf["#{env_name}_slave"] = slave_conf
+        if replica_conf = env_config.delete('slave')
+          # TODO
+          env_config['replica'] = replica_conf
```

Comment:
I think this should be the other way around. *If* anyone adds both `slave` and `replica` configuration (while migrating from the old format to the new format) I think we should let the new config (replica) take precedence.

Reply:
I agree with Benjamin here. We should give precedence to the `replica` key.